source/common/http/conn_manager_impl.h

```diff
@@ -279,7 +279,7 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
     Router::RouteConstSharedPtr route(const Router::RouteCallback& cb) override;
     void setRoute(Router::RouteConstSharedPtr route) override;
     void clearRouteCache() override;
-    absl::optional<Router::ConfigConstSharedPtr> routeConfig() override;
+    absl::optional<Router::ConfigConstSharedPtr> routeConfig();
```

Comment:
nit: If this is no longer an override, we should probably move it out of the section of methods which all come from a subclass?

Reply:
good call